viewer/apiHunts.js

```diff
@@ -412,7 +412,7 @@ ${Config.arkimeWebURL()}sessions?expression=huntId==${huntId}&stopTime=${hunt.qu
       return HuntAPIs.#huntFailedSessions(hunt, huntId, options, searchedSessions, user);
     }
 
-    Db.searchSessions(['sessions2-*', 'sessions3-*'], query, { scroll: internals.esScrollTimeout }, function getMoreUntilDone (err, result) {
+    Db.searchSessions(Db.defaultIndexPatterns(Config.getArray('queryExtraIndices', '')), query, { scroll: internals.esScrollTimeout }, function getMoreUntilDone (err, result) {
```

Comment:
Throughout the code the several places where `['sessions2-*', 'sessions3-*']` are passed into db.js have been replaced by `Db.defaultIndexPatterns` which takes an `extraIndices` array (pulled from the config's `queryExtraIndices` value) and merges its contents with the default `['sessions2-*', 'sessions3-*']` array. This merged array is returned.